CalibTracker/SiStripChannelGain/interface/SiStripGainsPCLWorker.h

```diff
@@ -93,15 +93,22 @@ class SiStripGainsPCLWorker : public  DQMEDAnalyzer {
 
     int statCollectionFromMode(const char* tag) const;
 
-    std::vector<MonitorElement*>  Charge_Vs_Index;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTIB;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTOB;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTIDP;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTIDM;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTECP1;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTECP2;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTECM1;
-    std::vector<MonitorElement*>  Charge_Vs_PathlengthTECM2;
+    std::vector<MonitorElement*>  Charge_Vs_Index;           /*!< Charge per cm for each detector id */
+    std::vector< std::vector<MonitorElement*> > Charge_1;    /*!< Charge per cm per layer / wheel */
```

Comment:
Hi @dimattia - as the inner vector length is hardwired to be 7, please make this a std::vector<std::array<MonitorElement*,7> >.